code/modules/shuttle/marine_dropship.dm

```diff
@@ -622,6 +618,9 @@
 
 /obj/machinery/computer/shuttle/marine_dropship/proc/do_hijack(obj/docking_port/mobile/marine_dropship/crashing_dropship, obj/docking_port/stationary/marine_dropship/crash_target/crash_target, mob/living/carbon/xenomorph/user)
 	crashing_dropship.set_hijack_state(HIJACK_STATE_CRASHING)
+	if(isdistress(SSticker.mode))
+		var/datum/game_mode/infestation/distress/distress_mode = SSticker.mode
+		distress_mode.round_stage = DISTRESS_DROPSHIP_CRASHING
```

Comment:
Why not do this with a signal for COMSIG_GLOB_DROPSHIP_HIJACKED

Reply:
But why? we are just changing a flag, no need for a signal